.github/workflows/create-branch-from-linear.yml

```
@@ -0,0 +1,100 @@
+        if: github.event_name != 'workflow_dispatch'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ISSUE_URL="${{ github.event.issue.url }}"
+          REPO_URL="https://github.com/${{ github.repository }}"
+          COMMENT_BODY="🚀 Branch [\`${BRANCH_NAME}\`](${REPO_URL}/tree/${BRANCH_NAME}) created from \`develop\`."
+          curl -s -X POST \
+            -H "Authorization: token ${GH_TOKEN}" \
+            -H "Content-Type: application/json" \
+            -d "{\"body\":\"${COMMENT_BODY}\"}" \
+            "$ISSUE_URL/comments"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)